tests/e2e/suite.go

```diff
@@ -0,0 +1,16 @@
+package e2e
+
+import (
+	"github.com/stretchr/testify/suite"
+
+	"github.com/cosmos/gaia/v23/tests/e2e/common"
+	"github.com/cosmos/gaia/v23/tests/e2e/msg"
+	"github.com/cosmos/gaia/v23/tests/e2e/tx"
+)
+
+type IntegrationTestSuite struct {
+	suite.Suite
+	commonHelper common.Helper
```

Comment:
What do you think about just embedding here?
Didn't check whether `tx` or `msg` are stateless, but if they are I'd suggest the same.

Reply:
Probably cleaner, will try that

Reply:
Common is stateful as it uses counters for proposals, code IDs, contracts deployed per address, etc.

I don't think I could could embed the structs without causing circular dependencies

Reply:
Nvm, I moved some things around and it seems to work - lmk what you think